### ISSUE TITLE
[1.3.2] PHP 7 Kompatibilität

### ISSUE THREAD

**Issue by baumisimon** [terminal42/contao-notification_center#86]
Folgender Fehler kommt beim Bearbeiten einer Benachrichtigung bei Einsatz von PHP 7 (final):
```
Fatal error: Cannot use NotificationCenter\Util\String as String because 'String' is a special class name in /system/modules/notification_center/library/NotificationCenter/Gateway/File.php on line 15
```

**Comment by aschempp** [comment]
We would need to drop support for Contao < 3.5 for a good fix. The only other option I can think of is a version_compare …

**Comment by Toflar** [comment]
Yes, we would need to make it 3.5.4+ only (which applies for all Contao extensions that want to be compatible with PHP7 and use the `String` class). Also it will be kind of a mess because we also have a `String` named util class :D

**Comment by aschempp** [comment]
Not sure if we actually use the Contao `String` class. If we rename and deprecate our class the problem should be gone from that side.

**Comment by Toflar** [comment]
We do, I checked.

**Comment by Mynyx** [comment]
Dies scheint der einzige Punkt zu sein, bei dem es für eine PHP 7 Kompatibilität von Isotope noch hängt?

**Comment by Toflar** [comment]
Hab die Liste aktualisiert, aber wie man an der Liste sieht, ist das längst nicht alles.

**Comment by aschempp** [comment]
@qzminski can you provide a PR for this (in regards to https://github.com/isotope/core/issues/1602)? I would suggest a `version_compare` like we do in all the other places already...

**Comment by Toflar** [comment]
However, this one is more complicated because we have our own `String` class that needs to be renamed :)

**Comment by aschempp** [comment]
Yeah, similar to Contao 3.2 right? We «just» need to add the new class and keep the BC for third-party extensions.

**Comment by qzminski** [comment]
Fixed in 8084e2b.
